src/store.cc

```diff
@@ -1009,6 +1005,10 @@ StoreEntry::checkCachable()
         } else if (fdNFree() < RESERVED_FD) {
             debugs(20, 2, "StoreEntry::checkCachable: NO: too many FD's open");
             ++store_check_cachable_hist.no.too_many_open_fds;
+        } else if (EBIT_TEST(flags, ENTRY_NEGCACHED)) {
+            debugs(20, 3, "StoreEntry::checkCachable: YES: negative cached");
+            ++store_check_cachable_hist.yes.negative_cached;
+            return 1;
```

Comment:
I'd be happy to do that, but similar to Alex's comments on the debug message, isn't it more important to have consistency in the function than to have one 'true' with all the other 1s and zeros returned in the function?  Would you like me to change them all?

Reply:
While I disagree with this change request (on the same vague consistency grounds), to save time and keep making progress, my recommendation is to change any touched return values. Unlike debugging/output changes in the other change request, here, it will address Amos requests while causing only mild inconsistencies.

Please do not change untouched `return` lines!

Reply:
This section has been removed.